source/speech/__init__.py

```diff
@@ -2442,6 +2440,8 @@ def getSpeechWithoutPauses(  # noqa: C901
 		L{GeneratorWithReturn} to retain the return value. A generator is used because the previous
 		implementation had several calls to speech, this approach replicates that.
 		"""
+		if speechSequence is not None:
+			logBadSequenceTypes(speechSequence)

```

Comment:
Since `getSpeechWithoutPauses` can not accept an generator, also perform the check here.